mkdocs.yml

```
@@ -39,6 +39,7 @@ nav:
 - Data:
   - HTTP & OASIS:
     - 'Install Squid HTTP Cache': 'data/frontier-squid.md'
+    - 'Running Frontier Squid in a Container': 'data/run-frontier-squid-container.md'
```

```suggestion
    - 'Running Frontier Squid in a Container': 'data/run-frontier-squid-container.md'
    - 'Install Frontier Squid RPM': 'data/frontier-squid.md'
```

I'd like to push folks towards the container so I'm putting this first. Adding as a suggestion since I'm too lazy to make a separate PR :)